### ISSUE TITLE
Consult the source code of GraphENS about  datasets Amazon 

### ISSUE THREAD

**Issue by yanliang3612** [JoonHyung-Park/GraphENS#1]
Dear JoonHyung Park：
I have one  question about GraphENS,I didn't find how the two datasets Amazon_computers and Amazon_photos are split on the source code of GraphENS, I read the paper that the number of each class is the same for the validation set and the test set, and the rest Is the training set, what is the number of each type of the specific validation set and test set, and can you provide me with some detailed separation.I would like to reproduce your experiment with this dataset from Amazon.I hope this  messages can get your reply, thanks！！！
